pandas/tests/io/formats/test_format.py

```diff
@@ -3262,6 +3262,11 @@ def test_format_percentiles():
     expected = ["0%", "50%", "2.0%", "50%", "66.67%", "99.99%"]
     assert result == expected
 
+    # Issue #46362
+    result = fmt.format_percentiles([0.281,0.57,0.58,0.29])
```

Comment:
You need spaces after the commas. Also, those are supposed to be percentiles, so they should be sorted.